### PR TITLE
ChecksumEthereumAddress_eip55

### PR DESCRIPTION
**Pull request by xxScorpius97xx**

### CODE REVIEW

File: Event_Timer/deploy.yml
Comment (xxScorpius97xx) [inline Event_Timer/deploy.yml:1]: Add deploy.yml for lagrange

**Review by xxScorpius97xx — COMMENTED** [review]
Add deploy.yml for lagrange

**Comment by Normalnoise** [comment]
please add a readme for event__timer

**Comment by xxScorpius97xx** [comment]
https://github.com/swanchain/awesome-swanchain/pull/165/commits/e705944203a210b5aa85156f8bbf1d4883f505ef

This one

**Comment by xxScorpius97xx** [comment]
Here
[Readme.md](https://github.com/user-attachments/files/16869196/Readme.md)

Next time i use branch 🙏🏼

**Comment by xxScorpius97xx** [comment]
The readme.md has already in the folder [Event_Timer](https://github.com/swanchain/awesome-swanchain/blob/main/Event-Timer)

e705944
1d63b66
a29ae7a
776163f
Those just to add deploy.yml to Event_Timer and fix broken link on your main README.md
https://github.com/swanchain/awesome-swanchain/blob/main/Event-Timer

Should i cancel all of those? And create new  request?


> please add a readme for event__timer

**Review by Normalnoise — APPROVED** [review]

**Pull request merged**